app/telegram/models.py

```diff
@@ -27,6 +27,8 @@ def _get_first_name(self, data: Dict) -> str:
 
 
 class Bot:
+    MEMORY = {}
```

Comment:
This is problematic. It's equivalent to a global variable and might create problems when working asynchronously.